src/mscorlib/shared/System/Numerics/Vector.cs

```diff
@@ -4953,6 +4954,7 @@ private static T GetAllBitsSetValue()
         #endregion
     }
 
+    [Intrinsic]
```

Comment:
It sounds like it would be good to remove this long term (and to just handle it these as normal named intrinsics), can we log a bug tracking that?

Reply:
(coming from https://github.com/dotnet/coreclr/pull/16133#discussion_r165207416).

Reply:
Isn't that already tracked by https://github.com/dotnet/corefx/issues/25386?

If you mean just this one `[Intrinsic]` attribute should be removed, I don't think it would make sense to handle just the `static class Vector` methods as normal named intrinsics, but not all the other `Vector<T>` methods.  If the JIT is the one accelerating `Vector<T>` methods, then it should be the one deciding if `IsHardwareAcceleration` is true or false.

Reply:
> Isn't that already tracked by dotnet/corefx#25386?

Not quite, #25386 tracks investigation on whether or not using Hardware Intrinsics instead of direct JIT support would be better.

> If you mean just this one [Intrinsic] attribute should be removed, I don't think it would make sense to handle just the static class Vector methods as normal named intrinsics, but not all the other Vector<T> methods.

I was specifically referring to the comment from @jkotas on:
> Ideally, we would change the JIT to recognize the IsHardwareAccelerated method as intrinsic without needing to mark the static Vector type as instrinsic.

I agree that getting them all fixed up would be desirable, but it seemed like this one in particular might have been directly beneficial (since it requires us to check all types, rather than just all generic value types).